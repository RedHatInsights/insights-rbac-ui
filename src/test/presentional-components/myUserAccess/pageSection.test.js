import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import MUAPageSection from '../../../presentational-components/myUserAccess/pageSection';

describe('<MUAPageSection />', () => {
  it('should render correctly', () => {
    expect(toJson(mount(
        <MUAPageSection
            title='test title'
            description='test description'>
            <span> test children </span>
        </MUAPageSection>
    ))).toMatchSnapshot();
  });
});
