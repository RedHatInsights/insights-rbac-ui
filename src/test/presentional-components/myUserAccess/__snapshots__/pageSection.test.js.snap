// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MUAPageSection /> should render correctly 1`] = `
<MUAPageSection
  description="test description"
  title="test title"
>
  <section
    className="ins-l-myUserAccess-section"
  >
    <Stack
      hasGutter={true}
    >
      <div
        className="pf-l-stack pf-m-gutter"
      >
        <StackItem>
          <div
            className="pf-l-stack__item"
          >
            <Title
              headingLevel="h2"
              size="2xl"
            >
              <h2
                className="pf-c-title pf-m-2xl"
              >
                test title
              </h2>
            </Title>
            <p>
               
              test description
            </p>
          </div>
        </StackItem>
        <StackItem>
          <div
            className="pf-l-stack__item"
          >
             
            <span>
               test children 
            </span>
             
          </div>
        </StackItem>
      </div>
    </Stack>
  </section>
</MUAPageSection>
`;
